js/supermarkets.js

```diff
@@ -17,25 +17,24 @@ function initMap() {
 }
 
 searchMarketsNear('München, Deutschland');
+clickmyBtn();
 
-/*Triggering Button when Enter is pressed */
-function triggerFunction() {
-    input.addEventListener("keyup", function(e) {
-        if (e.key === 'Enter') {
+/*Triggering Function when Enter is pressed */
+input.addEventListener("keyup", function(e) {
+        if (e.key === 'Enter' && input.value !== '') {
             searchMarketsNear(input.value);
         }
-    });
+});
+
+//Adding eventListener to my Button
+function clickmyBtn () {
```

Comment:
two things:
1. how it's different from adding event listener for input?
2. this is defenetely not a proper name for the function. Becasuse what it does is to add listener and not click a button.